### ISSUE TITLE
Req: Search bar Unicode support

### ISSUE THREAD

**Issue by Jonta** [stefanXO/Tab-Manager-Plus#93]
Reproduce:

1. [ddg-search "kümmer"](https://duckduckgo.com/?q=k%C3%BCmmer&t=ha&ia=web)
2. Open TMP
3. Search "kümmer" in any view
4. On "k", the tab is highlighted
5. But add "ü", and the tab is no longer highlighted

```
TMP-only Firefox-profile
TMP 5.1.6
Firefox 75.0
macOS 10.11.6
```

**Comment by mastef** [comment]
I don't think this is "unicode support" related. The extension normally works for japanese and other utf-8 characters.

![image](https://user-images.githubusercontent.com/762297/81407015-d7b5e280-9164-11ea-91ca-ecc440109a8b.png)

What happens if you type in `kümmer` and then press backspace? Or search for `ümmer`. Does it highlight the tabs for you? Are you typing in `ü` natively on your keyboard, or are you holding `u` long and then using the apple quickselect for special chars?

**Comment by Jonta** [comment]
Huh, indeed:

- Type "ümmer"
- No highlight
- Backspace
- Tab is highlighted

And I *do* press something else on my keyboard to get the ü-character (¨ and then u. Norwegian layout)

**Comment by mastef** [comment]
@Jonta  I think that's the issue. It happens on my mac as well when I make these special chars ( long-press `u` and then select `ü` ). I will have to take a look into this

**Comment by mastef** [comment]
Should be fixed in c70cf3e24cb2a70814febacacd87cf9d6bf7dacd and 5.2.0
